src/sentry/utils/pytest/selenium.py

```diff
@@ -181,6 +187,31 @@ def wait_until_not(self, selector=None, title=None, timeout=10):
     def switch_to(self):
         return self.driver.switch_to
 
+    def attempts(self, func, times=5):
+        """
+        Attempt the provided func n times until it succeeds.
+        This is useful when trying to resolve timeouts/flakey tests
+        in travis that are hard to reproduce locally.
+        """
+        retryable_errors = (
+            NoSuchElementException,
+            ElementNotVisibleException,
+            ElementNotSelectableException,
+            TimeoutException
+        )
+        attempt = 0
+        while attempt < times:
```

Comment:
Oops, we never increment `attempt`. It looks like CI basically timed out doing this and never succeeded, so the retry might not be helping at all.

Reply:
Good catch 👍

Reply:
Whoops 😊  If the retries aren't working, I wonder what the state of the database is, perhaps the event isn't being persisted?

Reply:
@markstory I was wondering the same thing when I tried to debug this locally. Perhaps the state hasn't been settled yet.

I'm wondering if there's some sort of django signal we can "wait for". 🤔